AdyenCard/Utilities/BinLookup/BinLookupService.swift

```diff
@@ -27,12 +27,19 @@ internal final class BinLookupService: AnyBinLookupService {
 
     private var cache = [String: BinLookupResponse]()
     
-    internal init(publicKey: String, apiClient: APIClientProtocol) {
+    private let binLookupType: BinLookupRequestType
+    
+    internal init(publicKey: String, apiClient: APIClientProtocol, binLookupType: BinLookupRequestType) {
         self.publicKey = publicKey
         self.apiClient = apiClient
+        self.binLookupType = binLookupType
     }
     
-    internal func requestCardType(for bin: String, supportedCardTypes: [CardType], caller: @escaping CompletionHandler) {
+    internal func requestCardType(
+        for bin: String,
+        supportedCardTypes: [CardType],
+        caller: @escaping CompletionHandler
```

Comment:
Not sure why it was called `caller` before but can we rename it to `completion` :D